titlingpage.tex

```diff
@@ -0,0 +1,23 @@
+\begin{titlingpage}
+\begin{center}
+\vspace*{\fill}%
+\makebox[\linewidth]{\HRule} % asks latex room & draws line
+\makeatletter
+\parbox[t]{\textwidth}{\addvspace{\parskip} %  create parbox for the text add space underline to parskip
+\centering\huge\bfseries\@title}
+ \null %0space so that latex doesn't ignore the parskip
+\vspace*{\parskip}
+\makebox[\linewidth]{\HRule}
+\vspace*{\fill}
+\ifx\@author\empty%
+\else%
+\par \Large Written by \@author
+\vspace*{\fill}
+\fi
+\ifx\@date\empty%
+\else%
+\par Compiled on \@date
+\fi
+\makeatother
+\end{center}
+\end{titlingpage}
```

Comment:
This is way more complicated than I'd like to include for a title page -- is there not a package that already does this / why would you prefer this over the default maketitle?

Reply:
This part is irrelevant. I'll discard it for now.